lib/screens/apply_task.dart

```diff
@@ -0,0 +1,179 @@
+                  RichText(
+                    text: TextSpan(
+                        style: TextStyle(color: Colors.orange, fontSize: 15),
+                        text: 'Address'),
+                  ),
+                  RichText(
+                    text: TextSpan(
+                        style: TextStyle(color: Colors.white, fontSize: 15),
+                        text: _task.formattedAddress),
+                  ),
+                  SizedBox(height: 20.0),
+                  RichText(
+                    text: TextSpan(
+                        style: TextStyle(color: Colors.orange, fontSize: 15),
+                        text: 'Date & Time'),
+                  ),
+                  RichText(
+                    text: TextSpan(
+                        style: TextStyle(color: Colors.white, fontSize: 15),
+                        text: _task.date.toString()),
+                  ),
+                  SizedBox(height: 20.0),
+                ],
+              ),
+            ),
+            SizedBox(
+              height: 60,
+            ),
+            Center(
+              child: Column(
+                children: <Widget>[
+                  RaisedButton(
+                    color: Colors.orange,
+                    textColor: Colors.white,
+                    child: Text('Interested!'),
+                    shape: RoundedRectangleBorder(
+                      borderRadius: new BorderRadius.circular(18.0),
+                    ),
+
+                    onPressed: () async {
+                      showDialog(
+                        context: context,
+                        builder: (BuildContext context) {
+                          return AlertDialog(
+                            title: Text("Show Interest"),
+                            content: Text("Show that you are interested in helping with the task"
+                                "the task provider will be notified and can choose to accept you help."
+                                "You will reviece a notification in the task provider has accepted your help."),
+                            actions: [
+                              FlatButton(
+                                onPressed: (){
+                                  Navigator.of(context).pop();
+                                },
+                                child: Text("close"),
+                              )
+                            ],
+                          );
+                        }
+                      );
+                      try {
+                        FirebaseUser user = await FirebaseAuth.instance.currentUser();
+
+                        print(_task.toString());
+                        
+                        await Firestore.instance.collection('tasks/${_task.id}/applicants').document(user.uid).setData({'status': 'interrested'});
+
+                        // _task.
+
+                        // await Firestore.instance.collection('tasks')
```

Comment:
same here, regarding to comments

Reply:
Remove comments and commit and create a new pull request